graphql_compiler/compiler/expressions.py

```diff
@@ -248,6 +257,8 @@ def get_local_object_gremlin_name(self):
     def validate(self):
         """Validate that the LocalField is correctly representable."""
         validate_safe_string(self.field_name)
+        if self.field_type is not None and not is_graphql_type(self.field_type):
+            raise ValueError(u'Invalid value of "field_type": {}'.format(self.field_type))
```

Comment:
nit: Add the field_name to this assertion error.

Reply:
I'll force it to write out the entire `LocalField` so we are guaranteed to have all the arguments, even if we add more in the future.